command/host_create_test.go

```diff
@@ -68,6 +70,30 @@ func (s *createHostSuite) TestParamDefaults() {
 	s.Equal(apimodels.DefaultContainerWaitTimeoutSecs, s.cmd.CreateHost.ContainerWaitTimeoutSecs)
 }
 
+func (s *createHostSuite) TestParseFromFile() {
+	//file for testing parsing from a json file
+	tmpdir, err := ioutil.TempDir("", "evergreen.command.host_create.test")
+	s.Require().NoError(err)
+
+	path := filepath.Join(tmpdir, "example.json")
+	fileContent := map[string]interface{}{
+		"distro":    "myDistro",
+		"scope":     "task",
+		"subnet_id": "${subnet_id}",
+	}
+
+	s.NoError(utility.WriteJSONFile(path, fileContent))
+	_, err = os.Stat(path)
+	s.Require().False(os.IsNotExist(err))
+	s.params = map[string]interface{}{
+		"file": path,
```

Comment:
Can you add "background" to this test to make sure it defaults correctly? You should also validate that createHost has the values you would expect.